asciidoc-confluence-publisher-doc/etc/docs/00-index/10-plantuml.adoc

```diff
@@ -43,3 +43,54 @@ plantuml::../files/included-diagram.puml[]
 ....
 
 plantuml::../files/included-diagram.puml[]
+
+== C4-PlantUML
+
+https://github.com/plantuml-stdlib/C4-PlantUML[C4-PlantUML] diagrams are supported as well.
+
+----
+[plantuml, c4-diagram, svg]
```

Comment:
Maybe you could shorten the example of the C4 model (e.g. to the one you listed on https://github.com/confluence-publisher/confluence-publisher/issues/310#issue-839351415) since in my opinion the documentation should just show how to integrate C4 models, but not necessarily show an extended example with many features (that I would expect to find in the linked C4 documentation).

Reply:
Agree. Made a quick change. Thanks.  :-)

Reply:
Now I am scared @wfouche - looks like you where doing exactly the above already while I was typing my comment ...

Reply:
We had the same idea at the same time.